### PR TITLE
[WIP] Wrapper of is_connected

### PR DESCRIPTION
**Pull request by OrkoHunter**
WIP

### CODE REVIEW

**Comment by OrkoHunter** [comment]
And that's our first green tick! Well that's because we've got 0 test cases.

**Comment by kshitij10496** [comment]
Yeah !
We need to add the test cases soon.

File: nxcpy/components/_connected.pyx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+__all__ = [
+        'connected_components',
+        'is_connected'
+]
+
+cdef enum Color:
+    WHITE = 0
+    GRAY = 1
+    BLACK = 2
+
```
Comment (kshitij10496) [inline nxcpy/components/_connected.pyx]: Rather than assigning numbers to each colors cant we just refer them via the color names ?

Comment (OrkoHunter) [inline nxcpy/components/_connected.pyx]: This will be removed. The enum from `bfs.h` will be wrapped and used.

Comment (kshitij10496) [inline nxcpy/components/_connected.pyx]: Cool.
If you need any help with this, I am free 
Waiting for the wrapper to be completed and for that first moment of satisfaction ! :smile:

Comment (OrkoHunter) [inline nxcpy/components/_connected.pyx]: Let me scratch my part of head first. :D

Comment (kshitij10496) [inline nxcpy/components/_connected.pyx]: Cool.
Just keep me updated !

**Comment by OrkoHunter** [comment]
Will continue after exams. Cheers!

```
    Among the dirty code of ours,
        There lies a fix.
```

**Comment by kshitij10496** [comment]
Cheers !

**Pull request closed without merging**